crowdstrikeoauthapi.json

```diff
@@ -16641,26 +17727,10 @@
                 "module": "beautifulsoup4",
                 "input_file": "wheels/py3/beautifulsoup4-4.9.1-py3-none-any.whl"
             },
-            {
```

Comment:
I think we can and should drop the `pip_dependencies` and only leave `pip39_dependencies`

Reply:
Done